enhancements/vdpa/virtio-vdpa-ovn-hw-offload.md

```diff
@@ -17,18 +17,19 @@ api-approvers:
   - "@dcbw"
   - "@knobunc"
 creation-date: 2022-07-13
-last-updated: 2022-07-20
-status: provisional
+last-updated: 2022-07-29
+tracking-link: 
+  - https://issues.redhat.com/browse/NP-19
```

Comment:
Are you tackling the whole of NP-19, or just a specific story with this commit?

Reply:
Yes, the plan is to tackle the whole NP-19 epic. The only thing is that the BF2 card is not explicitly mentioned in this enhancement, but only the CX-6 DX card to be supported in the first place. I'm not expecting big changes for integrating the BF2 card though, everything should be handled by the vdpa driver.

Reply:
Thanks for clarifying.